pkg/bmc/idrac_virtualmedia.go

```diff
@@ -86,3 +88,53 @@ func (a *redfishiDracVirtualMediaAccessDetails) RAIDInterface() string {
 func (a *redfishiDracVirtualMediaAccessDetails) VendorInterface() string {
 	return "no-vendor"
 }
+
+func (a *redfishiDracVirtualMediaAccessDetails) ValidateBootInterface(bmcCreds Credentials) (bool, error) {
+	var hasVirtualMedia, err = a.validateVirtualMedia(bmcCreds)
+	return hasVirtualMedia, err
+}
+
+func (a *redfishiDracVirtualMediaAccessDetails) validateVirtualMedia(bmcCreds Credentials) (bool, error) {
+	bmcConfig := gofish.ClientConfig{
+		Endpoint: getRedfishAddress(a.bmcType, a.host),
+		Username: bmcCreds.Username,
+		Password: bmcCreds.Password,
+		Insecure: !a.disableCertificateVerification,
+	}
+
+	client, err := gofish.Connect(bmcConfig)
+	if err != nil {
+		panic(err)
+	}
+	defer client.Logout()
+
+	service := client.Service
+
+	managers, err := service.Managers()
+	if err != nil {
+		panic(err)
+	}
+
+	for _, manager := range managers {
+		vmediaList, err := manager.VirtualMedia()
+		if err != nil {
+			return false, err
+		}
+		if vmediaList == nil {
```

Comment:
This returns an error if any item in the list of managers does not have the required support, but doesn't that mean we would not look at all of the managers? I think you should just `continue` here, right?

Reply:
Yeah, I will update this